README.md

```diff
@@ -62,14 +72,25 @@ __# useful Docker functions__
        sudo docker $1 $c
    done
 }
-* alias cd.....="cd ../../../.." 
-* alias cd......="cd ../../../../.." 
-
-* alias cd1="cd .." 
-* alias cd2="cd ../.." 
-* alias cd3="cd ../../.." 
-* alias cd4="cd ../../../.." 
-* alias cd5="cd ../../../../.." 
+__# Kubernetes commands__
+* alias k="kubectl"
+* alias kg="kubectl get"
+* alias kga="kubectl get all"
+* alias kgall="kubectl get all --all-namespaces"
+* alias ka="kubectl apply -f"
+* alias klo="kubectl logs -f"
+* alias kex="kubectl exec -i -t"
+
+__# Docker commands__
+* alias k="kubectl"
```

Comment:
Thanks for the PR. you can remove this line. also on lines 62 and 63, you can add `sudo` just like `sudo` has been used in its previous commands. It has been purposefully written. I can merge the PR after these minute changes.